<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTools</class>
 <widget class="QWidget" name="FormTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tools</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>Save as:</string>
   </property>
   <widget class="QPushButton" name="BtnPdf">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>140</width>
      <height>70</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Assets/Images/IMG-Pdf.png</normaloff>../Assets/Images/IMG-Pdf.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnWord">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>140</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Assets/Images/IMG-Word.png</normaloff>../Assets/Images/IMG-Word.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnCancel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{color:white;background-color: rgb(205, 97, 85);border:solid 1px black;}
QPushButton:hover{background-color: rgb(229, 152, 102);}
QPushButton:pressed{background-color: rgb(220, 118, 51);}
</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnSave">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{color:white;background-color: rgb(205, 97, 85);border:solid 1px black;}
QPushButton:hover{background-color: rgb(229, 152, 102);}
QPushButton:pressed{background-color: rgb(220, 118, 51);}
</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-3</x>
      <y>110</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
